src/lib/components/workflow/workflow-atom.svelte

```diff
@@ -0,0 +1,204 @@
+<script context="module" lang="ts">
+  import type { WorkflowExecution } from '$lib/types/workflows';
+
+  export type RootNode = {
+    children: RootNode[];
+    workflow: WorkflowExecution;
+  };
+</script>
+
+<script lang="ts">
+  import { page } from '$app/stores';
+
+  import Button from '$lib/holocene/button.svelte';
+  import Tooltip from '$lib/holocene/tooltip.svelte';
```

Comment:
Now that we can have more complex tooltips what if we put the rest of the electrons "tree" into a tooltip 🤔

Reply:
Hehe yes. But electrons all have the same tree. Maybe on node click we zoom in on node and show just it's subtree and hide/fade the rest